components/FloatingContact/src/index.scss

```diff
@@ -39,6 +39,13 @@
   z-index: var(--denhaag-floating-contact-index);
 }
 
+@media (min-width: 1024px) {
+  .denhaag-floating-contact {
```

Comment:
`.denhaag-floating-contact` ➝ `.denhaag-theme`

Reply:
@Robbert renamed it.